lib/mtl/rails/card_file_presenter.rb

```diff
@@ -0,0 +1,132 @@
+require 'uri'
+
+module Mtl
+  module Rails
+    # Specialised renderer for the file card element used in a file collection
+    #
+    # Basic usage:
```

Comment:
I guess the "public" API is `mtl_card_file`, right? So I propose to move this documentation to the `helpers.rb` and mark this one as private with a short description (one sentence) or what it's used for.